python/ql/src/change-notes/2022-10-10-pep249-executemany-modeling.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added model of `executemany` calls on PEP-249 compliant Database APIs, resulting in additional sinks for `py/sql-injection`.
```

Comment:
Is the uppercase D in "Database" significant?

Reply:
no.

```suggestion
* Added model of `executemany` calls on PEP-249 compliant database APIs, resulting in additional sinks for `py/sql-injection`.
```